README.md

```diff
@@ -1 +1,122 @@
-# integrity-sum
\ No newline at end of file
+# integrity-sum
+![GitHub contributors](https://img.shields.io/github/contributors/ScienceSoft-Inc/k8s-container-integrity-monitor)
+![GitHub last commit](https://img.shields.io/github/last-commit/ScienceSoft-Inc/k8s-container-integrity-monitor)
+![GitHub](https://img.shields.io/github/license/ScienceSoft-Inc/k8s-container-integrity-monitor)
+![GitHub issues](https://img.shields.io/github/issues/ScienceSoft-Inc/k8s-container-integrity-monitor)
+![GitHub forks](https://img.shields.io/github/forks/ScienceSoft-Inc/k8s-container-integrity-monitor)
+
+![Go](https://img.shields.io/badge/go-%2300ADD8.svg?style=for-the-badge&logo=go&logoColor=white)
+![Kubernetes](https://img.shields.io/badge/kubernetes-%23326ce5.svg?style=for-the-badge&logo=kubernetes&logoColor=white)
+![Postgres](https://img.shields.io/badge/postgres-%23316192.svg?style=for-the-badge&logo=postgresql&logoColor=white)
+![GitHub](https://img.shields.io/badge/github-%23121011.svg?style=for-the-badge&logo=github&logoColor=white)
+
+This program provides integrity monitoring that checks file or   directory of container to determine whether or not they have been tampered with or corrupted.  
+integrity-sum, which is a type of change auditing, verifies and validates these files by comparing them to the stored data in the database.
+
+If program detects that files have been altered, updated, added or compromised, it rolls back deployments to a previous version.
+
+integrity-sum injects a `hasher-sidecar` to your pods as a sidecar container. 
+`hasher-sidecar` the implementation of a hasher in golang, which calculates the checksum of files using different algorithms in kubernetes:
+* MD5
+* SHA256
+* SHA1
+* SHA224
+* SHA384
+* SHA512
+
+## Architecture
+
+## Getting Started
+
+### Clone repository and install dependencies
+```
+$ git clone https://github.com/ScienceSoft-Inc/integrity-sum.git
+$ cd path/to/install
```

Comment:
Should we add here `go mod download` to ensure that the user has needed libraries from `go.mod`?

Reply:
Fixed it.